src/Elastic.Apm/AgentComponents.cs

```diff
@@ -182,5 +187,24 @@ public void Dispose()
 
 			CentralConfigurationFetcher?.Dispose();
 		}
+
+		private static void PrintAgentLogPreambel(IApmLogger logger)
+		{
+			if (logger?.Info() != null)
+			{
+				var info = logger.Info().Value;
+				info.Log("********************************************************************************");
+				info.Log(
+					$"Elastic APM .NET Agent, version: {Assembly.GetExecutingAssembly().GetName().Version}, build date: {File.GetCreationTime(Assembly.GetExecutingAssembly().Location).ToUniversalTime()} UTC");
```

Comment:
The solution for the "build date" is not yet ideal, but a good first approximation.
A better mid- to long-term solution could be to introduce a custom assembly attribute that gets actually set a build time.

Reply:
👍  I also think this is a good first start. Just one thing maybe: I think it'd be better to print "agent file creation time" (or something along those lines) to make sure there is no misunderstanding by this log line.

Reply:
Agreed, then this will be unambiguous 👍